www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -75,8 +95,33 @@
   background-color: $c-divider;
 }
 
+.PreviewHeader {
+  @include f-note;
+  padding-top: 10px;
+  padding-bottom: 5px;
+}
+
 .PreviewBody {
   padding: 10px;
+
+  input {
```

Comment:
Added some very basics styles for the live code example output, just so it looks a bit more cared for (without looking like a "theme")